diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+grace$/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Github is a tool to store and organize git repositories.
```

Comment:
FYI, an important function of Git is version control. This means that we can make changes to our code, and it will update accordingly without keeping multiple copies of the same project.